release/release.js

```diff
@@ -47,7 +47,7 @@ const devBranchDefault = "dev2"
 const docBranchDefault = "gh-pages2"
 const versionReleasedDefault = VERSION
 const nextVersionDefault = "10.2.0"
-const versionCodeReleasedDefault = 77
+const versionCodeReleasedDefault = 79
```

Comment:
NB: we don't really have to update that file for every release , one can type in the correct value during the release, but that way there is less chance of errors.